devtools-ci-index.yaml

```diff
@@ -1630,6 +1631,13 @@
         <<: *vault_defaults
```

Comment:
Also. the `vault-secrets:` section is duplicate and is therefore unnecessary. The vault secrets are already defined in the `eclipse-che-automation-template`